docs/daemons.rst

```diff
@@ -0,0 +1,374 @@
+=======
+Daemons
+=======
+
+Daemons are a special type of handlers for background logic that accompanies
+the Kubernetes resources during their life cycle.
+
+Unlike event-driven short-running handlers declared with ``@kopf.on``,
+daemons are started when an object is created or an operator is started,
```

Comment:
This is a bit ambiguous: Is one task created per resource that the daemon triggers on or is only ever one task started when the first resource of that kind is created and stopped when the last resource of that kind is deleted?

Reply:
Paraphrased to this:

> Unlike event-driven short-running handlers declared with ``@kopf.on``,
> daemons are started for every individual object when it is created
> (or when an operator is started/restarted while the object exists),
> and are capable of running indefinitely (or infinitely) long.

Not sure if it is better though…

Reply:
It is, thanks.